tools/migrations/035_escha_portals_unlock.py

```diff
@@ -0,0 +1,27 @@
+import array
+import mysql.connector
+
+def migration_name():
+    return "Adding eschan_portals and vorseals columns to char_unlocks table"
```

Comment:
This says "and vorseals" but there's only one new column, are they combined or is this something leftover from the branch?

Reply:
oops, leftover. Originaly, In the Escha branch, vorseal logic was included. I decided to separate both.

Will fix now